etc/profile-m-z/parsecd.profile

```diff
@@ -0,0 +1,22 @@
+include parsecd.local
+include globals.local
+
+whitelist ${HOME}/.parsec
+whitelist /usr/share/parsec
+include whitelist-common.inc
+include whitelist-usr-share-common.inc
+
+apparmor
+nodvd
+nonewprivs
+nogroups
+notv
+nou2f
+noroot
+
```

Comment:
`seccomp` is a very powerful and important option for creating the sandbox. Even though I can feel your pain in having to go through the time-consuming work that's sometimes needed in this regard, it is what needs to be done :-) Take all the time you need for doing so and ask for help when something's not clear.

Reply:
This went much smoother than anticipated, only one syscall was missing: 243, aka tgkill and parsec handled that very gracefully.

Reply:
I noticed when ctrl+c'ing the application it complains of

```
mty_sleep: 'nanosleep' failed with errno 4
```

and in the journal:

```
Feb 11 17:59:16 archlinux kernel: audit: type=1300 audit(1676134756.097:498): arch=c000003e syscall=234 success=no exit=-13 a0=c a1=c a2=f a3=8 items=0 ppid=7627 pid=7639 auid=1000 uid=1000 gid=1000 euid=1000 suid=1000 fsuid=1000 egid=1000 sgid=1000 fsgid=1000 tty=pts3 ses=2 comm="parsecd" exe="/usr/local/bin/parsecd" subj=firejail-default//&unconfined key=(null)
Feb 11 17:59:16 archlinux kernel: audit: type=1400 audit(1676134756.097:498): apparmor="DENIED" operation="signal" profile="firejail-default" pid=7639 comm="parsecd" requested_mask="send" denied_mask="send" signal=term peer="firejail-default//&unconfined"
```

But i explicitely noblacklisted that syscall with !tgkill

Reply:
seccomp != AppArmor

Even if you all a syscall in the seccomp filter, AppArmor can still deny it because it is an independent of seccomp.

Reply:
How do I make my seccomp changes consistent with those AppArmor uses?